--- origsrc/binutils-2.25/bfd/elf64-sh64.c	2014-10-14 02:32:02.000000000 -0500
+++ src/binutils-2.25/bfd/elf64-sh64.c	2014-12-26 11:47:37.606929800 -0600
@@ -2259,31 +2259,10 @@ sh_elf64_set_private_flags (bfd *abfd, f
 static bfd_boolean
 sh_elf64_copy_private_data_internal (bfd *ibfd, bfd *obfd)
 {
-  Elf_Internal_Shdr **o_shdrp;
-  asection *isec;
-  asection *osec;
-
   if (bfd_get_flavour (ibfd) != bfd_target_elf_flavour
       || bfd_get_flavour (obfd) != bfd_target_elf_flavour)
     return TRUE;
 
-  o_shdrp = elf_elfsections (obfd);
-  for (osec = obfd->sections; osec; osec = osec->next)
-    {
-      int oIndex = ((struct bfd_elf_section_data *) elf_section_data (osec))->this_idx;
-      for (isec = ibfd->sections; isec; isec = isec->next)
-	{
-	  if (strcmp (osec->name, isec->name) == 0)
-	    {
-	      /* Note that we're not disallowing mixing data and code.  */
-	      if ((elf_section_data (isec)->this_hdr.sh_flags
-		   & SHF_SH5_ISA32) != 0)
-		o_shdrp[oIndex]->sh_flags |= SHF_SH5_ISA32;
-	      break;
-	    }
-	}
-    }
-
   /* Copy object attributes.  */
   _bfd_elf_copy_private_bfd_data (ibfd, obfd);
   
